build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -16,7 +16,7 @@
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
 
-    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461' AND '$(RuntimeIdentifier)' == '' AND '$(OS)' == 'Windows_NT'">win7</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461' AND '$(RuntimeIdentifier)' == '' AND '$(OS)' == 'Windows_NT'">$(RoslynDesktopRuntimeIdentifiers)</RuntimeIdentifier>
```

Comment:
Don't think we can use this property here. The `$(RoslynDesktopRuntimeIdentifiers)` is used in the `<RuntimeIdentifiers>` (emphasis on the plural form) position in the build files. This particular property is `<RuntimeIdentifier>` (the single form). If the `<RoslynDesktopRuntimeIdentifiers>` property ever switched to multi-targeting this would be invalid. 

Suggest that for now we change the `$(RoslynRuntimeIdentifier)` to a singular form.

Reply:
Is there a reason why we can't multitarget this instead? (change the prop to `RuntimeIdentifiers`). 

`VisualStudioInteractiveComponents.csproj` is also single-targeted.